### PR TITLE
Getting Products Data using API call on Mockbee Server

### PR DESCRIPTION
**Pull request by nareshpoonia**
Getting Products Data using API call on Mockbee Server

### CODE REVIEW

**Review by Hamdrive — COMMENTED** [review]
Liked the code format written, easy to read.
Good use of context logic, will be helpful as the project is being built

As a suggestion you can pass your props/state directly within the Link component of React-Router, thus you dont have to create separate routes for the same
Refer [here](https://twitter.com/_Diksha_Dhote/status/1508557213048373248)

File: src/pages/luggage.js [hunk 1]
```diff
@@ -2,245 +2,38 @@ import React from "react";
...
-            <div className="fs-8 m-8 flex justify-between">
-              <span>1</span>
-              <span>3</span>
-              <span>5</span>
-            </div>
-          </div>
-        </aside>
-        <main className="products-display grow">
-          <h1 className="ml-8 mt-8">Showing all products</h1>
+        <ProductsAside />
```
Comment (prachi-sahani) [inline src/pages/luggage.js]: This component can be named as "FilterSection" to signify the purpose of the component.

**Review by prachi-sahani — COMMENTED** [review]
+1 to whatever Hamza has mentioned about routes. Apart from that, the code is well structured and easily understandable.

File: src/pages/luggage.js (hunk 1 above)
Comment (nareshpoonia) [reply src/pages/luggage.js]: Thanks Prachi, change the route to a single products page & remanded the ProductAside component to FilterSection

**Pull request merged**